api.raml

```diff
@@ -231,32 +240,46 @@ schemas:
                 "status": 500
               }
 
-/user:
+/users:
```

Comment:
For the sake of consistency, I think we typically keep the endpoint singular, knowing that a `GET` request without an individual record identifier is a request for a list of all of them.